src/sempy_labs/_helper_functions.py

```
@@ -230,12 +239,26 @@ def delete_item(
     )
 
 
+@log
+def create_folder_if_not_exists(
+    folder: str | PathLike, workspace: Optional[str | UUID] = None
+) -> UUID:
+    try:
+        x = fabric.resolve_folder_id(folder=folder, workspace=workspace)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)